test_util/marathon.py

```diff
@@ -11,6 +11,7 @@
 
 TEST_APP_NAME_FMT = 'integration-test-{}'
 REQUIRED_HEADERS = {'Accept': 'application/json, text/plain, */*'}
+log = logging.getLogger(__name__)
```

Comment:
Why are we changing the logger for the python tests? Just wondering why its part of this PR?